### ISSUE TITLE
Accessibility: add aria-hidden=true to the dropdown icon

### ISSUE THREAD

**Issue by christiemolloy** [patternfly/patternfly-next#705]
Within the dropdown component, the caret down icon is missing aria-hidden=true. 

Add aria-hidden=true to the icon inside of the dropdown-toggle-icon hbs file.
